Makefile

```diff
@@ -120,6 +120,12 @@ _tmpdir-mk: ##@prepare Create a TMPDIR for temporary files
 # Make sure TMPDIR exists every time make is called
 -include _tmpdir-mk
 
+_install-hooks: SHELL := /bin/sh
+_install-hooks: ##@prepare Create prepare-commit-msg git hook symlink
+	@ln -s ../../scripts/hooks/prepare-commit-msg .git/hooks
+# Create prepare-commit-msg git hook symlink
```

Comment:
We don't need this duplicate comment, it's probably just some leftover above. What's important is the one above the target prefixed with `##@` which shows up in help normally.

Also, this target should probably be added to `.PHONY`.